build.sbt

```diff
@@ -1,35 +1,28 @@
-
 organization := "com.kashoo"
-
 name := """ws-limited"""
-
-version := "1.1.0"
-
+version := "4.0.0"
```

Comment:
Why such a big jump?

Reply:
It looks funny out of context and maybe I should change it to 2.0.0. The reason was that versions 2 and 3 were used by play 2.6 and 2.7, though not merged into master: https://github.com/Kashoo/ws-limited/blob/play-2.7.x/build.sbt.